Formula/ttab.rb

```diff
@@ -0,0 +1,14 @@
+class Ttab < Formula
+  desc "CLI for opening a new terminal tab/window"
+  homepage "https://github.com/mklement0/ttab"
+  url "https://raw.githubusercontent.com/mklement0/ttab/v0.6.1/bin/ttab"
+  sha256 "cdcbeb5ca8b95a2b85b68f4716a2748cd7d921bbb4d88786bc4a48031d5caf70"
+
+  def install
+    bin.install "ttab"
+  end
+
+  test do
+    system "#{bin}/ttab", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.